osf/migrations/0109_set_ever_public.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-06-08 15:01
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+def set_ever_public(state, *args, **kwargs):
+    (state.get_model('osf', 'preprintservice')
```

Comment:
Minor: Generally, prefer not breaking this way. When a line is too long (this wouldn't be), the preferred way is something like
```python
Thing.some_call(*arguments, **etc).some_other_call_that_makes_the_line_too_long(
    some_arg=x,
    another_arg=y,
    **etc
).additional_calls()
```